pipes/Broad_UGER/cluster-submitter.py

```diff
@@ -27,9 +37,17 @@
 if mem:
     # on UGER, memory requests are per-core (according to BITS as of Sept. 6, 2016)
     mem_per_core = round((int(mem)*1024)/int(cores), 2)
-    cmdline += ' -l h_vmem={}M,h_rss={}M '.format(  mem_per_core, round(1.2 * mem_per_core, 2)  )
+    if blacklisted_nodes:
+        cmdline += " -l h_vmem={}M,h_rss={}M,h='!({})' ".format(
```

Comment:
We should probably have comment lines here explaining the various resource parameters, or at least for `h=` mention that it's the hostname param, that it [accepts a regex](http://wiki.gridengine.info/wiki/index.php/EvaluationExpressionSupport), and we take advantage of that invert the match and blacklist hostnames.